src/Microsoft.TestPlatform.ObjectModel/TestCase.cs

```diff
@@ -221,7 +223,7 @@ private Guid GetTestId()
             // For example in the database adapter case this is not a file path.
             string source = this.Source;
 
-            if (File.Exists(source))
+            if (PlatformFile.Exists(source))
```

Comment:
IFileHelper.Exists: https://github.com/Microsoft/vstest/blob/master/src/Microsoft.TestPlatform.CoreUtilities/Helpers/Interfaces/IFileHelper.cs#L34

Reply:
I know, but I do not want to create a new Instance of FileHelper to check for file Exists. Correct thing to do would be to move it out of coreutilities, & make it a static class in platform abstraction, but that would require a lot of tests changes.

I'm still thinking on how to do it better.